bin/initdb.sh

```
@@ -0,0 +1,188 @@
+    RUNNING=0
+fi
+
+# Start PostgreSQL
+WinStrom_startIfNeeded $RUNNING
+
+# Fix port in configuration file (it can be changed by user). localhost:5434 is the default. If user changed it, we will not overwrite it.
+sed -i -r "s/(port\">)$OLD_PORT/\1$PG_PORT/g" /etc/abraflexi/abraflexi-server.xml
+
+# check if user 'dba' exists.
+DBEXIST=$(/bin/su - "$PG_USER" -c "/usr/bin/psql $CLUSTERARG -t -A --command \"select usename from pg_user where usename = 'dba'\" | grep dba > /dev/null 2>&1 && echo OK" || true)
+if [ "x$DBEXIST" != "xOK" ]; then
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)